fit/core-reference/src/main/resources/log.properties

```diff
@@ -0,0 +1,17 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Only, be sure that such file is not included by archetype, deb, standalone or any other artifact relying on fit/core-reference.
If possible, try to avoid adding such file.